### DISCUSSION TITLE
The order of element

### DISCUSSION THREAD

**Discussion by xiangyyu** [deepmodeling/deepmd-kit#1987] [Q&A]
Dear DP developer,

I am working on training a potential energy surface with four dataset, each consisting of a reaction. I am wondering if the order of element in each type_map.raw file should be the same. For example, they are all in an order of " C H N O", not "C H O N". Also should I keep the same order in the input.json file for training and LAMMPS input file for simulation?

Thanks!

**Comment by njzjz** [comment]
> I am wondering if the order of element in each type_map.raw file should be the same. 

No.

> Also should I keep the same order in the input.json file for training and LAMMPS input file for simulation?

Yes.

**Reply by xiangyyu** [reply]
Thanks for your help! I appreciate it!
